src/Goteo/Library/Forms/Model/ProjectCampaignForm.php

```diff
@@ -25,23 +25,20 @@
 
 class ProjectCampaignForm extends AbstractFormProcessor implements FormProcessorInterface {
 
-    public function getConstraints($field): array
+    public function getPhoneConstraints(): array
     {
-        $constraints = [];
-
-        if($field === 'paypal') {
-            $constraints[] = new Constraints\Email([
-                'checkMX' => true,
-                'checkHost' => true
-            ]);
-        }
-        if($this->getFullValidation()) {
-            if($field === 'phone') {
-                $constraints[] = new Constraints\NotBlank();
-            }
+        if ($this->getFullValidation()) {
+            return [new Constraints\NotBlank()];
+        } else {
+            return [];
         }
+    }
 
-        return $constraints;
+    public function getPaypalConstraints(): array
+    {
+        return [
+            new Constraints\Email(),
+        ];
```

Comment:
@davidbeig actually, I had already created a method that kept the `Constraints\Email` constraint.